clash/build.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(let_chains)]
+fn main() {
+    println!("cargo:rerun-if-env-changed=CARGO_PKG_VERSION");
```

Comment:
this wouldn't rebuild on master when only ref changes but not pkg version right?

Reply:
right, But for now it only cares package version. If someday we have extra requirements, it's not late to change it then.

Reply:
But you'd want to rebuild for every master commit when version doesn't change?

Reply:
Will rebuild happen when actions are traiggered? 

I am a little not sure for now, what should I change this into?

Reply:
Actually it should be fine. Any code change on master should trigger a rebuild anyways.